src/components/photoViewer/PhotoViewer.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+export const PhotoViewer: React.FunctionComponent = () => {
+    return (
+      <div>
+        PhotoViewer
+      </div>
+    );
+  };
+  
```

Comment:
The final line should be completely blank (including spaces! No characters of any kind), so each file should end with _exactly one_ blank line